pybricksdev/cli/__init__.py

```diff
@@ -165,7 +165,9 @@ def add_parser(self, subparsers: argparse._SubParsersAction):
 
     async def run(self, args: argparse.Namespace):
         from ..ble import find_device
-        from ..connections import EV3Connection, PybricksHub, REPLHub
+        from ..connection.ev3 import EV3Connection
+        from ..connection.pybricks import PybricksHub
+        from ..connection.repl import REPLHub
```

Comment:
I'm not sure what to call this module but instead of `repl` it should convey that this is a connection to a Powered Up hub running official LEGO firmware that runs MicroPython (SPIKE Prime, SPIKE Essential and MINDSTORMS Robot Inventor).

Reply:
Or maybe we just call the module `lego` and put all official LEGO firmware connection types in there. I think that would fit nicely with the `ev3dev` and `pybricks` categorizations.

Reply:
Done.